sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -197,10 +202,30 @@ public IdentityClientOptions setAllowUnencryptedCache(boolean allowUnencryptedCa
         return this;
     }
 
+    /**
+     * Allows to override the tenant being used in the authentication request
+     * via {@link com.azure.core.experimental.credential.TokenRequestContext#setTenantId(String)}.
+     *
+     * @param allowMultiTenantAuthentication the flag to indicate if multi tenant authentication is enabled or not.
+     * @return The updated identity client options.
+     */
+    public IdentityClientOptions setAllowMultiTenantAuthentication(boolean allowMultiTenantAuthentication) {
+        this.allowMultiTenantAuthentication = allowMultiTenantAuthentication;
+        return this;
+    }
+
+
     public boolean getAllowUnencryptedCache() {
         return this.allowUnencryptedCache;
     }
 
+    /**
+     * Get the flag indicating if multi tenant authentication is enabled or not.
+     *
+     * @return the boolean status indicating if multi tenant authentication is enabled or not.
+     */
+    public boolean getAllowMultiTenantAuthentication() { return this.allowMultiTenantAuthentication; }
```

Comment:
fix formatting to not have the method body in the same line.

Reply:
Do you remember why is this and the method above not following `isXxx` naming convention for boolean properties?